src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -216,6 +216,16 @@ public function testRegister()
         $this->assertInstanceOf(Definition::class, $builder->getDefinition('foo'), '->register() returns the newly created Definition instance');
     }
 
+    public function testRegisterChild()
+    {
+        $builder = new ContainerBuilder();
+        $builder->register('foo', 'Bar\FooClass');
+        $builder->registerChild('bar', 'foo');
+        $this->assertTrue($builder->hasDefinition('bar'), '->registerChild() registers a new service definition');
+        $this->assertInstanceOf(ChildDefinition::class, $definition = $builder->getDefinition('bar'), '->registerChild() returns the newly created ChildDefinition instance');
```

Comment:
the description of the assertion does not match was is asserted

Reply:
Fixed